src/main/java/fxlauncher/Launcher.java

```diff
@@ -248,12 +259,12 @@ private void startApplication() throws Exception {
 				appParams.getRaw().addAll(params.getRaw());
 				appParams.getUnnamed().addAll(params.getUnnamed());
 			}
-			PlatformImpl.setApplicationName(app.getClass());
+			//Platform.setApplicationName(app.getClass());
```

Comment:
Intentional change?  if so please remove uncommented code

Reply:
Removed as access to internal PlatformImpl is not allowed anymore. Didn't find a proper other implementation way but kept comment be able to support setting correct application name later

Reply:
I suggest you document this accordingly
```suggestion
                      // Removed as access to internal PlatformImpl is not allowed anymore. Didn't find a proper other implementation way but kept comment be able to support setting correct application name later
			//Platform.setApplicationName(app.getClass());
```